cnd/src/swap_protocols/rfc003/alice/actions/erc20.rs

```diff
@@ -64,8 +64,8 @@ where
             actions.push(Action::Redeem(<(BL, BA)>::redeem_action(
                 HtlcParams::new_beta_params(request, response),
                 htlc_location.clone(),
-                &*self.secret_source,
-                self.secret_source.secret(),
+                self.secret_source, // Alice uses this swaps seed to derive the secret.
```

Comment:
Why are we passing the secret and the source if we only need the source for deriving the secret (assuming your comment is correct)?

Reply:
That is the exact question that led me to write this comment, I can't remember the answer right now but clearly the addition of this comment has not made the code clear enough :)